### DISCUSSION TITLE
Typography and background colors not applicable in Tailwind css 3.3.0

### DISCUSSION THREAD

**Discussion by Revaycolizer** [tailwindlabs/tailwindcss#10893] [Help]
I had this project of nextjs that I implemented some of Typography including background color and border but after updating it to latest version all background colors disappeared and border also including configuring custom color in Tailwind config.js won't solve the problem, I tried initialize a new nextjs project and added tailwind but it did the same thing

**Comment by wongjn** [comment]
Consider providing a remotely-hosted git repository that reproduces your problem.

**Reply by Revaycolizer** [reply]
https://github.com/Revaycolizer/Nova-Music

**Reply by wongjn** [reply]
It is because you have completely overridden the default color palette provided by Tailwind. It seems like you may be after *adding* to the Tailwind colors, not *replacing* them. In which case, make the following change in your `tailwind.config.js`:

```diff
 theme: {
-  colors:{
-    'hov': '#F2DFF1'
-  },
-  extend: {},
+  extend: {
+    colors: {
+      hov: '#F2DFF1',
+    },
+  },
 },
```

**Reply by Revaycolizer** [reply]
Thanks it's working
